pkg/ddevapp/settings_test.go

```diff
@@ -82,3 +93,174 @@ func TestWriteDrushConfig(t *testing.T) {
 	err = os.RemoveAll(dir)
 	assert.NoError(t, err)
 }
+
+// TestIncludeSettingsDdevInNewSettingsFile verifies that when no settings.php file exists,
+// a settings.php file is created that includes settings.ddev.php.
+func TestIncludeSettingsDdevInNewSettingsFile(t *testing.T) {
+	dir := testcommon.CreateTmpDir("")
+	err := os.MkdirAll(filepath.Join(dir, "sites/default"), 0777)
```

Comment:
This doesn't seem to take the docroot into account? This should be docroot + sites/default.

Also, this is drupal only right? Probably should update the test name for that.